website/docs/components/copy/button/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,41 @@
+## Usage
+
+### When to use
+
+- To copy large text blocks, such as code snippets, API keys, or Tokens. 
+
+### When not to use
+
+- To copy the text within the button, use a [Copy Snippet](/components/copy/snippet).
+
+## Sizes
+
+The `Copy::Button` comes in small and medium sizes. Use the size that best fits the hierarchy of the UI. 
+
+<Doc::Layout @spacing="16px">
+  <Hds::Copy::Button @text="Copy" @size="small" />
+  <Hds::Copy::Button @text="Copy" @size="medium" />
+</Doc::Layout>
+
+## Placement 
+
+Don’t place the `Copy::Button` in a disconnected area within the UI. It should be visually associated with specific content or context to prevent user confusion.
+
+!!! Do
```

Comment:
I would reverse the order of the Do/Dont blocks so the incorrect example comes first, and then you show them how to correct it.